lang/cs/Org.Apache.REEF.Tang.Tests/ClassHierarchy/TestClassHierarchy.cs

```diff
@@ -189,10 +189,11 @@ public void TestNamedRepeatConstructorArgClasses()
         }
 
         [TestMethod]
-        public void TestResolveDependencies() 
+        public void TestResolveDependencies()
         {
-            ns.GetNode(typeof(SimpleConstructors).AssemblyQualifiedName);
-            Assert.IsNotNull(ns.GetNode(typeof(string).AssemblyQualifiedName));
+            ns = TangFactory.GetTang().GetClassHierarchy(new string[] { FileNames.Examples });
```

Comment:
You shouldn't need the `string[]` here, as the method takes / should take var args.